Assets/Code/src/Runtime/Match/Components/MatchManager.cs

```diff
@@ -51,16 +48,16 @@ public class MatchManager : MonoBehaviour {
     }
 
     Debug.Log("Starting cooldown...");
-    IsControllerRunning = false;
-    await Mediator.Global.PublishAsync(new MatchStartCountdownEvent
+	 MatchController.CurrentState.StateID = MatchStateID.Intro;
+	 await Mediator.Global.PublishAsync(new MatchStartCountdownEvent
     {
         MatchConfig = Config,
         MatchState = MatchController.CurrentState
     });
-    IsControllerRunning = true;
-
-    Debug.Log("Running match...");
-    MatchTask = new TaskCompletionSource<MatchResult>();
+		
+	 Debug.Log("Running match...");
+	 MatchController.CurrentState.StateID = MatchStateID.InGame;
+	 MatchTask = new TaskCompletionSource<MatchResult>();
```

Comment:
The style for this project uses spaces, please change the tabs to spaces instead.